src/librbd/cache/pwl/AbstractWriteLog.cc

```diff
@@ -1551,11 +1542,21 @@ bool AbstractWriteLog<I>::check_allocation(
       m_bytes_allocated += bytes_allocated;
       m_bytes_cached += bytes_cached;
       m_bytes_dirty += bytes_dirtied;
+      if (m_cache_state->clean && m_bytes_dirty) {
```

Comment:
Just a nit but this would read more natural to me:
```suggestion
      if (m_cache_state->clean && bytes_dirtied > 0) {
```

Reply:
FWIW this nit was less about `> 0` and more about using the local variable (i.e. `bytes_dirtied`) instead of `m_bytes_dirty` member here.